src/types/deployment.ts

```diff
@@ -20,7 +20,21 @@ export type ListDeploymentsParams = {
   users?: string;
 } & PaginationParameters;
 
-export interface DeploymentList {
+export type GetDeploymentEventParams = {
+  deploymentIdOrUrl: string;
+  builds: number;
+  delimiter: number;
+  direction: string;
+  follow: number;
+  limit: number;
+  name: string;
+  since: number;
+  statusCode: number | string;
+  teamId: string;
+  until: number;
```

Comment:
`limit` and `until` means these take pagination params so we have to use the pattern of `& PaginationParameters` here.